src/script/auth/page/HistoryInfo.tsx

```diff
@@ -70,10 +76,10 @@ const HistoryInfoComponent = ({
   return (
     <Page>
       <ContainerXS centerText verticalCenter style={{width: '100%'}}>
-        <H1 center>{_(headline, {brandName: Config.getConfig().BRAND_NAME})}</H1>
+        <H1 center>{_(historyInfoStrings.noHistoryHeadline, {brandName: Config.getConfig().BRAND_NAME})}</H1>
         <Paragraph center style={{marginBottom: 56}}>
           <FormattedMessage
-            {...infoText}
+            {...historyInfoStrings.noHistoryInfo}
```

Comment:
im a bit confused by this change. if you have history shouldnt it show a different text?

Reply:
The thing is: it's now impossible to see this screen and have an actual history. This is due to past limitations that were deprecated since https://github.com/wireapp/wire-webapp/pull/7610

Reply:
sharing links from pre-covid 😂. approved then.

Reply:
Yeah, I'm digging into some old stuff in there 😆